proto/etcd.proto

```diff
@@ -36,11 +36,16 @@ message ResponseHeader {
     int64 revision = 3;
     // raft_term is the raft term when the request was applied.
     uint64 raft_term = 4;
+
+    // revision of the store that has most recently been committed (been through consensus).
+    int64 committed_revision = 5;
```

Comment:
Given the split consistency model (reads served off of the historical index vs reads served off the optimistic maps) I think it would be valuable to have both fields. This helps keep the client informed in both scenarios and allows them to switch their queries easily without having to query the revision independently.

Reply:
Seems like this might be nice for benchmarking, Maybe merge this and if performance is below expectations we can revisit

Reply:
I'm happy for these fields to be included in the response header. As @heidihoward said, we can always revisit this later.